### ISSUE TITLE
[Accessibility]A11y_VSCode_VSCDestinationLocation_Screenreader : Screenreader does not read the content on the screen in proper logical order.

### ISSUE THREAD

**Issue by AccessibilityTestingTeam-TCS** [Microsoft/vscode#51847]
**Environment Details:**
VSCode Version : 1.24.1
OS Version : Win10
**Additional Details:**
NVDA 2018.2, MAS1.3.2

**Repro Steps:**
1..On Visual Studio Code License Agreement screen,select "I accept the agreement" radiobutton and click "Next" button.
2.Check whether screenreader reads the content in proper logical order or not.

**Actual:**
Screenreader does not read the content on the screen in proper logical order. It reads At least 208.7 MB of free disk space is required after Setup will install Visual Studio Code into the following folder & then again it reads repetitive content.

**Expected:**
Screenreader should read the content in proper logical order as well as it should not read the same content twice.

**Recommendations:**
Refer below link which is repository of bug fixes code snippets:
https://microsoft.sharepoint.com/teams/msenable/mas/pages/browse-fixes.aspx

**User Impact:**
The user will get confused when it does not read the content in proper order.

**MAS Reference:**
https://microsoft.sharepoint.com/:w:/r/teams/msenable/_layouts/15/WopiFrame.aspx?sourcedoc={8492c4eb-c179-40ae-8777-cd044ed725a2}

**Does this issue occur when all extensions are disabled?: Yes

Attachment for Reference:**
![accessibility a11y_vscode_vscdestinationlocation_screenreader](https://user-images.githubusercontent.com/39204909/41396979-47770fb2-6fd0-11e8-9b90-57cd6aa35a3a.png)

**Comment by joaomoreno** [comment]
@AccessibilityTestingTeam-TCS This needs to go into https://github.com/jrsoftware/issrc
